README.md

```diff
@@ -34,102 +32,104 @@ _/bricks/lean-welcome.html_:
 
+_Pretty neat!_ <!-- FIXME: What is it related to?  -->
 
 ## Getting Started in 3 Minutes
 
-You can just hop in and experiment your first HTML Element.
-
-[Follow this Getting Started guide](https://lego.js.org/v2/getting-started/#quick-start) for installing and writing your first component.
-
-Once you're done with it you can write more components the same way and play with [daily usage guide](https://lego.js.org/v2/usage-web-components/) for passing advanced arguments, writing conditions and loops, dynamic styling, connecting to APIs…
+You can just hop in and experiment your first LEGO component. [Follow this "Getting Started" guide](https://lego.js.org/v2/getting-started/#quick-start) for installing and writing it. <!-- FIXME: Fix the documentation link -->
 
+Once you're done with it you can write more components the same way and play with [daily usage guide](https://lego.js.org/v2/usage-web-components/) for passing advanced arguments, writing conditions and loops, dynamic styling, connecting to APIs… <!-- FIXME: Fix the documentation link -->
 
 ## What Can You **Actually** Do With It?
 
-You can create a simple reusable web-components for your HTML website or build a full fledge app with it.
-It's as light as powerful, fully driven by the web-browsers standards.
-
-Various applications are running Lego in production for years now.
-
-Because of the standards and the zero dependency you can rely on it for the long term.
+You can create a simple reusable Web Components for your HTML website or build a full fledge app with it. It's as light as powerful, fully driven by the web-browsers standards.
 
-Your site/app will still running and remain editable in 10+ years, and as long as web browsers will exist.
+Various applications are running LEGO in production for years now. Because of the standards and the zero dependency you can rely on it for the long term. Your site/app will still run and remain editable in 10+ years, and as long as web browsers will exist.
 
 
 ## How Does LEGO Work?
 
-LEGO (_Lightweight Embedded Gluten-free Objects_) is a base class that you extend and provide you with methods to use ♻️ reactive and 🏡 **native web-components** that are easy to digest 🌱 for your browser.
+LEGO (_Lightweight Embedded Gluten-free Objects_) is a base class that you can extend and provide you with methods to use ♻️ reactive and 🏡 **native Web Components** that are easy to digest 🌱 for your browser.
 
 A small CLI in NodeJS come along to build 🚀 your component.
-There's no [Vite](https://vitejs.dev/) nor other 3rd parties, it's [just an interpreter of ±180 lines of readable code](https://github.com/Polight/lego/blob/master/src/compiler/transpiler.js).
-I also works with [https://bun.sh](Bun) and probably other interpreters.
+There's no [Vite](https://vitejs.dev/) nor other 3rd parties, it's [just an interpreter of ±130 lines of readable code](https://github.com/Polight/lego/blob/master/src/compiler/transpiler.js).
```

Comment:
I have updated the interpreter LOC, summing `vdom-parser` and `transpiler` files.